### PR TITLE
[pinmux] D0 -> D1 Review

### PR DESCRIPTION
**Pull request by msfschaffner**
This proposes to move the pinmux to D1. I'll add the reviewer names once it is clear who signs this off.

Signed-off-by: Michael Schaffner <msf@google.com>

### CODE REVIEW

File: hw/ip/pinmux/doc/checklist.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+
+[DV_PLAN_DRAFT_COMPLETED]:            {{<relref "/doc/project/checklist.md#dv-plan-draft-completed" >}}
+[TESTPLAN_COMPLETED]:                 {{<relref "/doc/project/checklist.md#testplan-completed" >}}
+[TB_TOP_CREATED]:                     {{<relref "/doc/project/checklist.md#tb-top-created" >}}
+[PRELIMINARY_ASSERTION_CHECKS_ADDED]: {{<relref "/doc/project/checklist.md#preliminary-assertion-checks-added" >}}
+[TB_ENV_CREATED]:                     {{<relref "/doc/project/checklist.md#tb-env-created" >}}
+[RAL_MODEL_GEN_AUTOMATED]:            {{<relref "/doc/project/checklist.md#ral-model-gen-automated" >}}
+[TB_GEN_AUTOMATED]:                   {{<relref "/doc/project/checklist.md#tb-gen-automated"
+>}}
```
Comment (tjaychen) [inline hw/ip/pinmux/doc/checklist.md]: @eunchan asked me to remove this newline also. We should probably get it fixed in the template.

File: hw/ip/pinmux/doc/checklist.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+Issues        | [ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED][] | Not Started |
+Integration   | [PRE_VERIFIED_SUB_MODULES_V2][]         | Not Started |
+Review        | [V3_CHECKLIST_SCOPED][]                 | Not Started |
+Review        | Reviewer(s)                             | Not Started |
+Review        | Signoff date                            | Not Started |
+
+
+[DESIGN_DELTAS_CAPTURED]:             {{<relref "/doc/project/checklist.md#design-deltas-captured" >}}
+[DV_PLAN_COMPLETED]:                  {{<relref "/doc/project/checklist.md#dv-plan-completed"
+>}}
```
Comment (tjaychen) [inline hw/ip/pinmux/doc/checklist.md]: same here with the newline?
@eunchan unless they're here for some reason?

File: hw/ip/pinmux/doc/checklist.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+--------------|---------------------------------------|-------------|------------------
+Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | In Progress |
+Documentation | [TESTPLAN_COMPLETED][]                | In Progress |
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [TB_ENV_CREATED][]                    | Done        |
+Testbench     | [RAL_MODEL_GEN_AUTOMATED][]           | N/A         | This block uses FPV
+Testbench     | [TB_GEN_AUTOMATED][]                  | N/A         | This block uses FPV
+Tests         | [SANITY_TEST_PASSING][]               | Done        |
+Tests         | [CSR_MEM_TEST_SUITE_PASSING][]        | N/A         | This block uses FPV
```
Comment (cindychip) [inline hw/ip/pinmux/doc/checklist.md]: I think similar to DV CSR test, FPV can have the register read/write assertions (I am still working on this in RV_PLIC ..)

File: hw/ip/pinmux/doc/checklist.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+[MISSING_FUNC]:        {{<relref "/doc/project/checklist.md#missing-func" >}}
+[FEATURE_FROZEN]:      {{<relref "/doc/project/checklist.md#feature-frozen" >}}
+[FEATURE_COMPLETE]:    {{<relref "/doc/project/checklist.md#feature-complete" >}}
+[AREA_SANITY_CHECK]:   {{<relref "/doc/project/checklist.md#area-sanity-check" >}}
+[DEBUG_BUS]:           {{<relref "/doc/project/checklist.md#debug-bus" >}}
+[PORT_FROZEN]:         {{<relref "/doc/project/checklist.md#port-frozen" >}}
+[ARCHITECTURE_FROZEN]: {{<relref "/doc/project/checklist.md#architecture-frozen" >}}
+[REVIEW_TODO]:         {{<relref "/doc/project/checklist.md#review-todo" >}}
+[STYLE_X]:             {{<relref "/doc/project/checklist.md#style-x" >}}
+[STYLE_LINT_SETUP]:    {{<relref "/doc/project/checklist.md#style-lint-setup" >}}
```
Comment (eunchan) [inline hw/ip/pinmux/doc/checklist.md]: Scott is about to removed this link

File: hw/ip/pinmux/doc/checklist.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+Review        | Signoff date            | Not Started |
+
+[NEW_FEATURES]:        {{<relref "/doc/project/checklist.md#new-features" >}}
+[BLOCK_DIAGRAM]:       {{<relref "/doc/project/checklist.md#block-diagram" >}}
+[DOC_INTERFACE]:       {{<relref "/doc/project/checklist.md#doc-interface" >}}
+[MISSING_FUNC]:        {{<relref "/doc/project/checklist.md#missing-func" >}}
+[FEATURE_FROZEN]:      {{<relref "/doc/project/checklist.md#feature-frozen" >}}
+[FEATURE_COMPLETE]:    {{<relref "/doc/project/checklist.md#feature-complete" >}}
+[AREA_SANITY_CHECK]:   {{<relref "/doc/project/checklist.md#area-sanity-check" >}}
+[DEBUG_BUS]:           {{<relref "/doc/project/checklist.md#debug-bus" >}}
```
Comment (eunchan) [inline hw/ip/pinmux/doc/checklist.md]: ditto

File: hw/ip/pinmux/doc/checklist.md (hunk 1 above)
Comment (eunchan) [reply hw/ip/pinmux/doc/checklist.md]: I thought I removed it... hum.. don't know why it still exist here..

Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: ok thanks, amended.

File: hw/ip/pinmux/doc/checklist.md (hunk 2 above)
Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: looks like these lines have been broken at the 80 char ruler... I'll amend.

File: hw/ip/pinmux/doc/checklist.md (hunk 3 above)
Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: Ok, I'll change to "In Progress"

File: hw/ip/pinmux/doc/checklist.md (hunk 2 above)
Comment (eunchan) [reply hw/ip/pinmux/doc/checklist.md]: @sjgitty already has a pending PR #912 in the template. please fix the doc only.

File: hw/ip/pinmux/doc/checklist.md (hunk 4 above)
Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: ok I'll remove

File: hw/ip/pinmux/doc/checklist.md (hunk 5 above)
Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: jup

**Comment by msfschaffner** [comment]
Ok thanks for the quick round of comments, I amended the parts you flagged. And added you all as reviewers for this transition.

**Comment by msfschaffner** [comment]
yeah I meant I'll fix the doc. I did not change the template.

File: hw/ip/pinmux/doc/checklist.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+--------------|---------------------------------------|-------------|------------------
+Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | In Progress |
+Documentation | [TESTPLAN_COMPLETED][]                | In Progress |
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [TB_ENV_CREATED][]                    | Done        |
+Testbench     | [RAL_MODEL_GEN_AUTOMATED][]           | N/A         | This block uses FPV
+Tests         | [SANITY_TEST_PASSING][]               | Done        |
+Tests         | [CSR_MEM_TEST_SUITE_PASSING][]        | In Progress | FPV CSR tests being developed by @cindychip
+Tool Setup    | [ALT_TOOL_SETUP][]                    | Done        |
```
Comment (weicaiyang) [inline hw/ip/pinmux/doc/checklist.md]: nit: since we only use JG, this should be N/A

File: hw/ip/pinmux/doc/checklist.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+Coverage      | [FUNCTIONAL_COVERAGE_V2][]              | Not Started |
+Issues        | [NO_HIGH_PRIORITY_ISSUES_PENDING][]     | Not Started |
+Issues        | [ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED][] | Not Started |
+Integration   | [PRE_VERIFIED_SUB_MODULES_V2][]         | Not Started |
+Review        | [V3_CHECKLIST_SCOPED][]                 | Not Started |
+Review        | Reviewer(s)                             | Not Started |
+Review        | Signoff date                            | Not Started |
+
+
+[DESIGN_DELTAS_CAPTURED]:             {{<relref "/doc/project/checklist.md#design-deltas-captured" >}}
```
Comment (sjgitty) [inline hw/ip/pinmux/doc/checklist.md]: the name changed here with #912, please revise

File: hw/ip/pinmux/doc/checklist.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+[NO_HIGH_PRIORITY_ISSUES_PENDING]:    {{<relref "/doc/project/checklist.md#no-high-priority-issues-pending" >}}
+[ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED]:{{<relref "/doc/project/checklist.md#all-low-priority-issues-root-caused" >}}
+[PRE_VERIFIED_SUB_MODULES_V2]:        {{<relref "/doc/project/checklist.md#pre-verified-sub-modules-v2" >}}
+[V3_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v3-checklist-scoped" >}}
+
+### Checklists for milestone V3
+
+ Type         | Item                              | Resolution  | Note/Collaterals
+--------------|-----------------------------------|-------------|------------------
+Documentation | [DESIGN_DELTAS_CAPTURED_IF_ANY][] | Not Started |
```
Comment (sjgitty) [inline hw/ip/pinmux/doc/checklist.md]: this one too.

File: hw/ip/pinmux/doc/checklist.md (hunk 6 above)
Comment (msfschaffner) [reply hw/ip/pinmux/doc/checklist.md]: jup changed. thanks.

**Comment by msfschaffner** [comment]
Thanks, amended.

**Pull request merged**